contrib/board_files/example.conf

```diff
@@ -0,0 +1,21 @@
+# A board config file consists of two required sections, the "board" and "GPIO"
+# Lines starting with "#" are ignored
+
+[board]
+# The dtfile is the file probed to determine if current board matches this
+# board config. Change this value if you system requires a different file for
+# board identification:
+# dtfile = /proc/device-tree/model
+
```

Comment:
# When attempting an automatic configuration probe at library runtime, the dtfile option contains the path
# to the file which will be read and it's contents compared against the 'model' option. Change this value if you want # your 'model' option to be compared to a different file.